test/integration/util/util.go

```diff
@@ -33,6 +33,8 @@ import (
 	commonutil "k8s.io/minikube/pkg/util"
 )
 
+const kubectlBinary = "kubectl-v1.6.0-alpha.1"
```

Comment:
to maintain compatibility with the current unit tests until we merge into master.

kubernetes 1.6 requires a kubectl 1.6